bundles/org.openhab.binding.systeminfo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -107,7 +107,7 @@
 		<channels>
 			<channel id="name" typeId="name"/>
 			<channel id="description" typeId="description"/>
-			<channel id="load" typeId="cpuLoad"/>
+			<channel id="load" typeId="load"/>
```

Comment:
Breaking change. Is it necessary to rename this channel type ?

Reply:
It would indeed be a breaking change, but the channel only got introduced invthe previous binding. And when calling it load, it can be used in multiple channel groups, cpuLoad does not fit. It is more consistent with the other channels used in multiple groups.

Reply:
When you write "the channel only got introduced in the previous binding", you mean when and in what OH version ?

Reply:
I mean, in the previous PR, and that is in the current milestone, but not release version.

Reply:
Ok, I found the PR, I merged it in September 11. So only users who installed recent 3.4 milestones could be impacted. I remove the tag.